internal/rego/matchers.go

```diff
@@ -89,15 +90,19 @@ func getKindMatchers(comment string) ([]KindMatcher, error) {
 
 func getMatchLabelsMatcher(comment string) (MatchLabelsMatcher, error) {
 	matcher := make(map[string]string)
-
 	matcherText := strings.TrimSpace(strings.SplitAfter(comment, "@matchlabels")[1])
-
 	for _, token := range strings.Fields(matcherText) {
 		split := strings.Split(token, "=")
 		if len(split) != 2 {
 			return nil, fmt.Errorf("invalid @matchlabels annotation token: %s", token)
 		}
+
 		matcher[split[0]] = split[1]
 	}
+
 	return matcher, nil
 }
+
+func commentHasKeyword(comment string, keyword string) bool {
+	return strings.HasPrefix(strings.TrimSpace(comment), keyword)
+}
```

Comment:
Added as a small helper to check for the existence of comment keywords such as `@enforcement`, `@kinds`, etc. so that this type of check is now done consistently throughout the codebase.